docusaurus.config.js

```diff
@@ -2,7 +2,7 @@
 // Note: type annotations allow type checking and IDEs autocompletion
 
 const lightCodeTheme = require("prism-react-renderer/themes/github");
-const darkCodeTheme = require("prism-react-renderer/themes/dracula");
+const darkCodeTheme = require("prism-react-renderer/themes/vsDark");
```

Comment:
Know that we have proper support for Clarity 2, I didn't like the way the dracula theme displays native function (in _italic_). So here's a test for vsDark.
It looks nice on my macbook pro but less nice on an other display I have (background can be a bit greenish). But developers are used to this background.

before/after
<img width="1013" alt="Screenshot 2023-02-21 at 00 55 33" src="https://user-images.githubusercontent.com/911307/220215405-dc361e34-b22c-46c4-ac3a-ef9c2c6d5ff2.png">
<img width="998" alt="Screenshot 2023-02-21 at 00 55 18" src="https://user-images.githubusercontent.com/911307/220215402-9960de91-3ef9-44e8-a762-f4f5ff011f3c.png">

Let me know what you think, I'm happy switching back to dracula as well

Reply:
I have no opinion to be honest - it's a cosmetic change so if someone doesn't like it we can address in a different issue. the change here seems fine to me, visually it still looks fine as well.